Models/Pokemon.swift

```diff
@@ -10,31 +10,31 @@ import SwiftUI
 import CoreLocation
 
 struct Pokemon: Hashable, Codable, Identifiable {
-    var id: String
-    var number: Int
+    var id: Int
+    var number: Int?
     var name: String
     var spritePC: String
-    var sprites: [String]
-    var pkType: [PokemonType]
-    init(id: String, number: Int, name: String, spritePC: String, pkType: [PokemonType]){
+    var sprites: [String]?
```

Comment:
The JSON data didn't have a sprites key, that is why I put it optional here, or it would crash parsing the json with Codable.